web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -74,20 +78,22 @@ function getTemplates() {
 
 let todayDate = new Date()
 const FeedbackRequestPage = () => {
+  const { state } = useContext(AppContext);
+  const { csrf} = state;
   const steps = getSteps();
   const classes = useStyles();
   const location = useLocation();
+  const { search } = useLocation();
+  const values = queryString.parse(search);
   const history = useHistory();
-
   const query = queryString.parse(location?.search);
-
   const stepQuery = query.step?.toString();
-
   let sendDate = query?.sendDate ? query.sendDate: todayDate.toString();
   let dueDate = query?.dueDate ? query.dueDate: null
-  let activeStep = location?.search ? parseInt(stepQuery) : 1;
-  const numbersOnly = /^\d+$/.test(stepQuery);
-
+  const [requestee, setRequestee] = useState();
+  const id = values.for;
+  let activeStep = location?.search ? parseInt(stepQuery) : 2;
+  //const numbersOnly = /^\d+$/.test(stepQuery);
```

Comment:
This commented line can be deleted, since we no longer use regex to validate the `stepQuery`